development.ini

```diff
@@ -24,6 +24,10 @@ available_languages =
     ara_AE
     rus_RU
     kir_KG
+    bur_MM
+    pt_BR
```

Comment:
We're using three letter language codes for the other languages. Just checking that its okay to be using a two-letter code here?